fprime_fpp/fprime_fpp_install.py

```diff
@@ -244,15 +245,38 @@ def safe_extract(tar, path=".", members=None, *, numeric_owner=False):
                 str(installation_directory),
             ],
         ]
+        # FPP install
         for step in steps:
             print(f"-- INFO  -- Running { ' '.join(step) }")
             completed = subprocess.run(step, env=subprocess_environment)
             if completed.returncode != 0:
                 print(f"-- ERROR -- Failed to run { ' '.join(step) }", file=sys.stderr)
                 sys.exit(-1)
-
+        # FPL install (https://github.com/fprime-community/fprime-layout)
+        if (os.getenv("FPL_NO_INSTALL") != "TRUE"):
+            install_fpl_via_git(installation_directory, subprocess_environment)
```

Comment:
FPL install has to be ran within the tools_directory context manager
https://github.com/fprime-community/fpp/blob/22dc4207987e0f30ac8c75a6c91fd9fd14ccb3e7/fprime_fpp/fprime_fpp_install.py#L214
since SBT is downloaded in this ephemeral directory.
So I am running the FPL install here, within the FPP install. This makes it a simpler PR, however another easy (and maybe cleaner?) solution would be to move the context manager up at this level: 
https://github.com/fprime-community/fpp/blob/22dc4207987e0f30ac8c75a6c91fd9fd14ccb3e7/fprime_fpp/fprime_fpp_install.py#L186
and run those "independently" (instead of nesting FPL within FPP) like such:
```
except urllib.error.HTTPError:
  with tempfile.TemporaryDirectory() as tools_directory:
    install_fpp_via_git(working_dir, tools_directory)
    install_fpl_via_git(working_dir, tools_directory)
```

I went with the PR that introduces the less changes. But let me know what you think, and I can quickly make the change if needed.

Reply:
This makes sense. I am in favor of the cleaner solution. I'm not too concerned about minimizing changes.